src/validation.cpp

```diff
@@ -2953,8 +2954,8 @@ bool Chainstate::FlushStateToDisk(
             TRACEPOINT(utxocache, flush,
                    int64_t{Ticks<std::chrono::microseconds>(SteadyClock::now() - nNow)},
                    (uint32_t)mode,
-                   (uint64_t)coins_count,
-                   (uint64_t)coins_mem_usage,
```

Comment:
This tracepoint was previously recording an estimate of memory usage of the entire cache, now it is an estimate of the disk usage of the dirty entries that were just written. Is that what we want to record here?

cc @0xB10C

Reply:
Hmm now we need to only declare `coins_mem_usage` if building trace points.